cpp_grpc_mt_bench/main.cpp

```diff
@@ -109,7 +109,7 @@ class ServerImpl final {
         new CallData(service_, cq_);
 
         // The actual processing.
-        reply_.set_message(request_.name());
+        *reply_.mutable_response() = request_.request();
```

Comment:
is it an implicit copy? if yes, can we avoid it?

Reply:
We can try `std::move`